learn.qmd

```diff
@@ -1,13 +1,73 @@
+| ---------------------------------------------- | ----------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Version Control with Git in Rstudio            | [{{< fa regular window-maximize title="Website" >}}](https://epiverse-trace.github.io/git-rstudio-basics/)  | [{{< fa brands github title="GitHub Repository" >}}](https://github.com/epiverse-trace/git-rstudio-basics/) | [{{< fa solid person-chalkboard title="Instructor view">}}](https://epiverse-trace.github.io/git-rstudio-basics/instructor/index.html)  | [![Static Badge](https://img.shields.io/badge/alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development)     |
+| Improve your code for Epidemic Analysis with R | [{{< fa regular window-maximize title="Website" >}}](https://epiverse-trace.github.io/research-compendium/) | [{{< fa brands github title="GitHub Repository" >}}](https://github.com/epiverse-trace/research-compendium) | [{{< fa solid person-chalkboard title="Instructor view">}}](https://epiverse-trace.github.io/research-compendium/instructor/index.html) | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+
+Para **Tutoriales en Español** recomendamos la [curricula del Software Carpentry en Español](https://software-carpentry.org/lessons/) con tutoriales sobre La terminal de Unix, Control de versiones con R, y R para análisis cientificos reproducible
+
+## Your questions
+
+Are you interested in participating in any of our workshops? Sign up for [our mailing list](https://data.org/initiatives/epiverse/our-community/), or follow us on Twitter at [Epiverse_TRACE](https://twitter.com/Epiverse_TRACE). You can also subscribe to [our blog feed](https://epiverse-trace.github.io/blog.html) and [take a look at our events calendar](https://view.monday.com/1267767805-fb40bb51c837164bf95532907829c3f8?r=euc1).
+
+Are you interested in teaching these materials? For this and more questions about broader training topics, [start a discussion](https://github.com/orgs/epiverse-trace/discussions/new/choose).
+
+For questions about a specific learning material, go to its repository to [create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue).
```

Comment:
From a chat with @lgbermeo, we found it valuable to add the list of workshops to this page. This will provide an idea to the readers of the amplitude and territoriality of the Epiverse-TRACE initiative and the possibilities of organising one of these locally. We expect to keep adding elements to this list after each training event. 

Also, as discussed in #193, we can optionally add links here to specific blog posts (like linking to these entries in [English](https://data.org/news/one-year-on-of-epiverse-trace-lac-exploring-the-data-science-and-health-ecosystem-in-latin-america-and-the-caribbean/) and [Spanish](https://data.org/news/un-ano-de-epiverse-trace-lac-explorando-el-ecosistema-de-ciencia-de-datos-y-salud-en-america-latina-y-el-caribe/))

```suggestion
For questions about specific learning material, go to its repository to [create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue).

# Our workshops

We used some of these materials in the following workshops:

- Curso Internacional: Análisis de Brotes y Modelamiento en Salud Pública https://epimodelac.com/, Bogotá (Colombia), December 4-8, 2023.
- Improving the reliability, usability and sustainability of code for epidemic analysis with R packages. IDDconf 2023, Ambleside, Cumbria, (UK), September 5-7, 2023.
- Curso-Taller: Herramientas de análisis y visualización de datos epidemiológicos para el reporte y la toma de decisiones en salud pública, Quibdó (Colombia), July 26-28, 2023.
- Curso-Taller: Teoría y práctica de respuesta a brotes y modelamiento matemático de epidemias, Bucaramanga (Colombia), April 27-29, 2023.
- Introductory training on Git and GitHub basics using Rstudio. London (UK), December 5, 2022.
- Taller de estudio de un brote y aplicación de la guía de estudio de brotes, Cali (Colombia), November 11-12, 2022.  
- Taller de Introducción a análisis de datos epidemiológicos en R y RStudio, Bogotá (Colombia),  October 24-25, 2023.
- Taller de Introducción al modelamiento y analítica de brotes y epidemias para informar la respuesta en salud pública, Bogotá (Colombia), October 24-25, 2023.
- Analizando un modelo básico de Zika, Cali (Colombia), October 15-16, 2022.
- Curso práctico en parámetros y modelos epidemiológicos en la respuesta a brotes y epidemias, Manizales (Colombia), September 1-2, 2022.
```

Reply:
Could you please track this in a separate issue? I implemented the copy as suppliedl; this is new copy, which will need to be reviewed separately. Adding that review into this PR extends the time needed to merge this PR (and the mental space needed to keep track of all the discussions).